front/app/component-library/components/Tooltip/index.tsx

```diff
@@ -17,16 +19,33 @@ const useActiveElement = () => {
     setActive(document.activeElement);
   };
 
+  const handleOutsideClick = () => {
+    setActive(null);
+  };
+
   useEffect(() => {
     document.addEventListener('focusin', handleFocusIn);
     return () => {
       document.removeEventListener('focusin', handleFocusIn);
     };
   }, []);
 
+  useEffect(() => {
```

Comment:
Fix for the tooltip component so that it would hide on outside click when an element inside the tooltip has been focused.